proxy/backend/core/src/test/resources/conf/convert/config-sharding.yaml

```diff
@@ -93,6 +96,10 @@ rules:
      props:
        algorithm-expression: t_order_item_${order_id % 2}
        datetime-lower: "2022-01-01 00:00:00"
+   t_order_item_hint_inline:
+     type: HINT_INLINE
+     props:
+       algorithm-expression: t_order_item_${order_id % 2}
```

Comment:
Property name in `HINT_INLINE` should be `value`, there is no column name in the strategy.

Reply:
FYI: "t_order_$->{value % 4}"

Reply:
got it